### PR TITLE
Convert SimpleFilter class component to functional

### PR DESCRIPTION
**Pull request by cdmh219**
Conversion of SimpleFilter class component to functional component, along with updating of unit tests. 

## Additions

- SimpleFilter functional component
- SimpleFilter functional component unit tests (using RTL)
- `data-testid` value in CollapsibleFilter component (with updating of snapshots)

## Removals

- SimpleFilter class component
- SimpleFilter class component unit tests
- SimpleFilter class component test snapshot

## Changes

-

## Testing

-

## Notes

-

## Todos

-

## Checklist

* [ ] Changes are limited to a single goal (no scope creep)
* [ ] Code can be automatically merged (no conflicts)
* [ ] Code follows the standards laid out in the [front end playbook](https://github.com/cfpb/front-end)
* [ ] Passes all existing automated tests
* [ ] New functions include new tests
* [ ] New functions are documented (with a description, list of inputs, and expected output)
* [ ] Placeholder code is flagged
* [ ] Visually tested in supported browsers and devices
* [ ] Project documentation has been updated (including the "Unreleased" section of the CHANGELOG)

### CODE REVIEW

**Review by flacoman91 — CHANGES_REQUESTED** [review]

File: src/components/Filters/CollapsibleFilter.js [hunk 1]
```diff
@@ -53,7 +53,7 @@ export default class CollapsibleFilter extends React.Component {
       </>
     );
     return (
-      <section className={composeClasses}>
+      <section className={composeClasses} data-testid="collapse-filter">
```
[the remark this answers is not in the document]
Comment (cdmh219) [reply src/components/Filters/CollapsibleFilter.js]: This is been taken out, and the related snapshots have been updated to not include it

File: src/components/Filters/SimpleFilter/SimpleFilter.spec.js [hunk 2]
```diff
@@ -0,0 +1,56 @@
...
+import SimpleFilter, { extraData } from './SimpleFilter';
+
+describe('component:SimpleFilter', () => {
+  describe('initial state', () => {
+    const props = { title: 'nana', fieldName: 'company_response' };
+
+    test('renders without crashing', () => {
+      render(<SimpleFilter {...props} />);
+      //first assertion is for ensuring that proper class is passed from SimpleFilter to child component
+      expect(screen.getByTestId('collapse-filter')).toHaveClass(
```
[the remark this answers is not in the document]
Comment (cdmh219) [reply src/components/Filters/SimpleFilter/SimpleFilter.spec.js]: This has been taken out. However, I have not updated the Cypress tests at this point, as the screen object is with the React Testing Library, which usually doesn't work with Cypress.

File: src/components/Filters/SimpleFilter/SimpleFilter.js [hunk 3]
```diff
@@ -0,0 +1,50 @@
...
+  const { options, hasChildren } = extraData(fieldName, aggs, query);
+
+  const listComponentProps = { fieldName };
+
+  return (
+    <CollapsibleFilter
+      title={title}
+      desc={desc}
+      hasChildren={hasChildren}
+      className={'aggregation simple ' + fieldName}
```
Comment (flacoman91) [inline src/components/Filters/SimpleFilter/SimpleFilter.js:33]: I don't think we need this

File: src/components/Filters/SimpleFilter/SimpleFilter.js [hunk 4]
```diff
@@ -0,0 +1,50 @@
...
+import AggregationItem from '../AggregationItem/AggregationItem';
+
+//TODO: when upgraded to redux toolkit, use createSelector
+//to derive this data directly from aggs and query state
+export const extraData = (fieldName, aggs, query) => {
+  const activeChildren = coalesce(query, fieldName, []);
+  const options = coalesce(aggs, fieldName, []);
+  const hasChildren = activeChildren.length > 0;
+
+  return { options, hasChildren };
```
Comment (flacoman91) [inline src/components/Filters/SimpleFilter/SimpleFilter.js:17]: can this go directly into SimpleFilter below?

File: src/components/Filters/SimpleFilter/SimpleFilter.spec.js [hunk 5]
```diff
@@ -0,0 +1,53 @@
...
+        foo: [1, 2, 3, 4, 5, 6],
+      };
+
+      query = {
+        foo: [1],
+      };
+    });
+
+    test('shows if there are any active children', () => {
+      expect(extraData(props.fieldName, aggs, query)).toEqual({
```
Comment (flacoman91) [inline src/components/Filters/SimpleFilter/SimpleFilter.spec.js:39]: can we render the component and then test for these items?

File: src/components/Filters/SimpleFilter/SimpleFilter.js (hunk 3 above)
Comment (cdmh219) [reply src/components/Filters/SimpleFilter/SimpleFilter.js:24]: The fieldname has been taken out of the className attribute

File: src/components/Filters/SimpleFilter/SimpleFilter.js (hunk 4 above)
Comment (cdmh219) [reply src/components/Filters/SimpleFilter/SimpleFilter.js]: This has been updated and properly verified in unit tests

**Review by flacoman91 — APPROVED** [review]

**Pull request merged**